client/landing/stepper/declarative-flow/migration-signup.ts

```diff
@@ -42,6 +40,7 @@ const migrationSignup: Flow = {
 			STEPS.PROCESSING,
 			STEPS.SITE_MIGRATION_UPGRADE_PLAN,
 			MIGRATION_INSTRUCTIONS_STEP,
+			STEPS.SITE_MIGRATION_STARTED,
```

Comment:
This and the new `case` are to fix the navigation to the Migration Started step on `migration-signup` flow.